pages/common/deemix.md

```diff
@@ -2,7 +2,7 @@
 
 > A barebone deezer downloader library built from the ashes of Deezloader Remix.
 > It can be used as a standalone CLI app or implemented in an UI using the API.
-> More information: <https://deemix.app>.
+> More information: <N/A>.
```

Comment:
Wow, nothing?

Reply:
There is <https://pypi.org/project/deemix/>, but there isn't anything on how to use it on there.

Reply:
Yeah, it's really odd.  Here's the old one, I guess: https://notabug.org/RemixDevs/DeezloaderRemix

Reply:
What does this link have to do with `deemix`?

Reply:
Hrm. I can't remember which issue it was we are discussing a not-applicable syntax, but this is incompatible with CommonMark as that is the autolinking syntax. Anyway, I'd suggest that <https://pypi.org/project/deemix/> is probably the best we've got to go with here - it's better than nothing at least.

Reply:
This site is only about the python package and doesn't include any information about how to use `deemix`. Do you think linking to a  site, where you dig a while just to find out there's nothing useful, is better than no link at all?

Reply:
Ah yeah, fair point. Let's do `More information: N/A.` then.

Reply:
There is this https://deezloader.net/deemix/

But it doesn't look very official and maybe breaking copyright and everything under the sun, it seems to have cracked in some way premium APKs however this may be the real site

Reply:
Hrm. In the interests of getting this PR merged, I suggest we remove the more information line completely from this page for now, and then when the linter is updated we can add the explicit `N/A` later.
```suggestion
```